src/configs/aggregator_config.cc

```diff
@@ -15,6 +15,23 @@ AggregatorConfig::AggregatorConfig(const Json::Value& json)
     this->prefix = this->prefix + ".";
   }
 
+  this->prefixCounter = json.get("prefix_counter", "counters").asString();
```

Comment:
Thanks for taking care of the config's backward compatibility by setting defaults.